database/schema/init_schema.sh

```diff
@@ -12,9 +12,9 @@ if $PG_INITIALIZED; then
   psql -d ${POSTGRESQL_DATABASE} -f ${CONTAINER_SCRIPTS_PATH}/start/create_users.sql
   # Create schema from scratch
   psql -d ${POSTGRESQL_DATABASE} -f ${CONTAINER_SCRIPTS_PATH}/start/create_schema.sql
-else
-  echo "Already initialized - Migrating the database"
-  ${CONTAINER_SCRIPTS_PATH}/migrate.sh up
+#else
```

Comment:
why do we comment that?

Reply:
Discussed personally - db migration does not work currently. It should be fixed.